lib/config/options/index.ts

```diff
@@ -42,7 +43,7 @@ const options: RenovateOptions[] = [
   {
     name: 'postUpgradeTasks',
     description:
-      'Post-upgrade tasks that are executed before a commit is made by Renovate.',
+      'Post-upgrade tasks that are executed before a commit is made by Renovate.', // What's the difference between `postUpgradeTasks` and the `commands` below?
```

Comment:
```suggestion
      'Post-upgrade tasks that are executed before a commit is made by Renovate.',
```

commands are a subset of postUpgradeTasks, they exist "inside" it